src/redux/Graphql/saga.js

```diff
@@ -45,13 +46,17 @@ export function* getTopicsHandler() {
         limit: action.limit,
         skip: action.skip,
       });
-    } catch (err) {
-      console.error(err); // eslint-disable-line
+    } catch (error) {
+      console.error(error); // eslint-disable-line
       yield put({
         type: actions.GET_TOPICS_RETURN,
         value: [],
         limit: action.limit,
         skip: action.skip,
+        error: {
```

Comment:
remove this error obj.

reason:
let's not throw an error for graphql calls that are called in the background (only for ones that require user interaction ie. button click). 

this would pop up every time if they went to the dashboard and could be really annoying if they have an error. graphql calls that call in the bg should fail silently and we can catch this error with remote error logging on UI side.